pkgs/by-name/dx/dxvk/package.nix

```diff
@@ -31,8 +31,8 @@ stdenvNoCC.mkDerivation (finalAttrs:
         --subst-var-by bash ${bash} \
         --subst-var-by dxvk32 ${dxvk32} \
         --subst-var-by dxvk64 ${dxvk64} \
-        --subst-var-by mcfgthreads32 "${pkgsCross.mingw32.windows.mcfgthreads_pre_gcc_13}" \
-        --subst-var-by mcfgthreads64 "${pkgsCross.mingwW64.windows.mcfgthreads_pre_gcc_13}"
+        --subst-var-by mcfgthreads32 "${pkgsCross.mingw32.windows.mcfgthreads}" \
+        --subst-var-by mcfgthreads64 "${pkgsCross.mingwW64.windows.mcfgthreads}"
```

Comment:
https://github.com/NixOS/nixpkgs/pull/302187 drops mcfgthreads from DXVK completely.

Reply:
Oh, nice. Didn't see that PR.

Reply:
I just pushed the fix tonight to the 2.3.1 update PR.